gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,124 @@
 ---
-description: Proof Market front end guide
+description: Web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
+
+The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
```

Comment:
Link formatting is enough to highlight the link from the surrounding text, so there's no need for bold.
```suggestion
The web interface of the Proof Market can be accessed at [proof.market](https://proof.market).
```

Reply:
Fixed